cmd/soroban-cli/src/commands/tx/add_op/args.rs

```diff
@@ -0,0 +1,46 @@
+use super::super::xdr::add_op;
+use crate::{
+    config::{address, locator},
+    xdr,
+};
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Address(#[from] address::Error),
+    #[error(transparent)]
+    TxXdr(#[from] super::super::xdr::Error),
+}
+
+#[derive(Debug, clap::Args, Clone)]
+#[group(skip)]
+pub struct Args {
+    #[clap(flatten)]
+    pub locator: locator::Args,
+    /// Source account used for the operation
+    #[arg(
+        long,
+        visible_alias = "op-source",
+        env = "STELLAR_OPERATION_SOURCE_ACCOUNT"
+    )]
+    pub operation_source_account: Option<address::Address>,
```

Comment:
Do you think we need a separate option for choosing the source account of operations? When adding an operation, there's no new tx being built, so the `--source-account` option should be available to use for this here, and the consistency will be more intuitive.